assets/js/modules/analytics-4/datastore/tags.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * `modules/analytics-4` data store: tags.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { createExistingTagStore } from '../../../googlesitekit/data/create-existing-tag-store';
+import { STORE_NAME } from './constants';
+import { tagMatchers } from '../utils/tag-matchers';
+import { isValidWebDataStreamID } from '../utils/validation';
+
+const existingTagStore = createExistingTagStore( {
+	storeName: STORE_NAME,
+	tagMatchers,
+	isValidTag: isValidWebDataStreamID,
```

Comment:
This shouldn't rely on `isValidWebDataStreamID`, the tag is the "measurement ID" - which is also stored in the web data stream entity, but it's not the web data stream ID.